RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelDigisClustersFromSoA.cc

```diff
@@ -120,7 +120,7 @@ void SiPixelDigisClustersFromSoA::produce(edm::StreamID, edm::Event& iEvent, con
 #ifdef EDM_ML_DEBUG
       ++totClustersFilled;
       LogDebug("SiPixelDigisClustersFromSoA")
-          << "putting in this cluster " << ic << " " << cluster.charge() << " " << cluster.pixelADC().size();
+          << "putting in this cluster " << ic << " " << acluster.charge << " " << acluster.isize;
```

Comment:
its ok for the debug. Sorry fpr the mess

Reply:
BUT need to be moved before the clear...

Reply:
```diff
diff --git a/RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelDigisClustersFromSoA.cc b/RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelDigisClustersFromSoA.cc
index 4c7eec701d3..634e4082ca7 100644
--- a/RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelDigisClustersFromSoA.cc
+++ b/RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelDigisClustersFromSoA.cc
@@ -113,14 +113,16 @@ void SiPixelDigisClustersFromSoA::produce(edm::StreamID, edm::Event& iEvent, con
                                                        << " size/charge " << acluster.isize << '/' << acluster.charge;
       // sort by row (x)
       spc.emplace_back(acluster.isize, acluster.adc, acluster.x, acluster.y, acluster.xmin, acluster.ymin, ic);
+#ifdef EDM_ML_DEBUG
+      LogDebug("SiPixelDigisClustersFromSoA")
+          << "putting in this cluster " << ic << " " << acluster.charge << " " << acluster.isize;
+#endif
       aclusters[ic].clear();
       std::push_heap(spc.begin(), spc.end(), [](SiPixelCluster const& cl1, SiPixelCluster const& cl2) {
         return cl1.minPixelRow() < cl2.minPixelRow();
       });
 #ifdef EDM_ML_DEBUG
       ++totClustersFilled;
-      LogDebug("SiPixelDigisClustersFromSoA")
-          << "putting in this cluster " << ic << " " << acluster.charge << " " << acluster.isize;
 #endif
     }
     nclus = -1;
```

like this?

Reply:
yes. Once can move ++totClustersFilled; as well.